src/Features/Core/Diagnostics/AbstractHostDiagnosticUpdateSource.cs

```diff
@@ -114,7 +114,7 @@ private void ClearAnalyzerDiagnostics(DiagnosticAnalyzer analyzer, ProjectId pro
 
         private DiagnosticsUpdatedArgs MakeArgs(DiagnosticAnalyzer analyzer, ImmutableHashSet<DiagnosticData> items, Project project)
         {
-            var id = WorkspaceAnalyzerManager.GetUniqueIdForAnalyzer(analyzer);
+            var id = analyzer.GetUniqueIdForAnalyzer();
```

Comment:
Minor: Probably rename to GetUniqueId now that its an extension on the analyzer?